BMM.Core/Translation/en/main.json

```diff
@@ -18,7 +18,8 @@
     "BackgroundActivityRestrictedTitle": "Allow background activity",
     "BackgroundActivityRestrictedMessage": "Your device settings restrict BMM from working in the background. You will not be able to listen, when your phone is locked for a few minutes. To change it, go to application settings and \"Allow background activity\" under the \"Battery\" section.",
     "GoToSettings": "Go to Settings",
-    "NotNow": "Not now"
+    "NotNow": "Not now",
+    "EnableWifiToContinue": "The playlist will be downloaded the next time you're on Wifi. You can enable \"Download via mobile network\" in your Profile."
   },
```

Comment:
I get the urge to put the Podcast message in PodcastViewModel. But I think in this case it's more important to have the Playlist message and the Podcast message close together.

Reply:
By the way, I think I told someone 3 years ago to do the same change. But he only changed it for podcasts and wasn't aware that it also needs to be done for Playlists.